Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -117,13 +119,17 @@ protected bool TestObjectExists(StorageService service, string bucket, string ob
     /// <example>
     ///   <para>Upload a local log file to GCS.</para>
     ///   <para><code>PS C:\> New-GcsObject -Bucket "widget-co-logs" -ObjectName "log-000.txt" `</code></para>
-    ///   <para><code>    -File "C:\logs\log-000.txt"</code></para>
+    ///   <para><code>    -Path "C:\logs\log-000.txt"</code></para>
     /// </example>
     /// <example>
     ///   <para>Pipe a string to a a file on GCS. Sets a custom metadata value.</para>
     ///   <para><code>PS C:\> "Hello, World!" | New-GcsObject -Bucket "widget-co-logs" -ObjectName "log-000.txt" `</code></para>
     ///   <para><code>    -Metadata @{ "logsource" = $env:computername }</code></para>
     /// </example>
+    /// <example>
+    ///   <para>Upload a folder to GCS</para>
```

Comment:
End the sentence with a period. But I'd suggest you elaborate on it. "Upload a folder and its contents to GCS. The names of the created objects will be relative to the folder."